helm_deploy/hmpps-welcome-people-into-prison-api/values.yaml

```diff
@@ -31,6 +31,8 @@ generic-service:
   namespace_secrets:
     hmpps-welcome-people-into-prison-api:
       APPINSIGHTS_INSTRUMENTATIONKEY: "APPINSIGHTS_INSTRUMENTATIONKEY"
+      SYSTEM_CLIENT_ID: "SYSTEM_CLIENT_ID"
```

Comment:
Not sure if we need to generate new credentials for the API or if we can just steal the UI ones?

Reply:
New ones really. Keep them segregated so usage can be tracked easier.

Reply:
might be worth talking this over tomorrow - we were considering proxying the request, so could potentially pass the system credentials through from the frontend service 🤔